### ISSUE TITLE
Exception when trying to get skin from URL [1.18]

### ISSUE THREAD

**Issue by DareFox** [TheDutchMC/SkinFixer#21]
When trying to get skin from URL, SkinFixer throws an exception

```
[11:00:00] [Server thread/INFO]: DareFox issued server command: /skin get https://i.imgur.com/4KQeMFe.png
[11:00:00] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'skin' in plugin SkinFixer v1.7.2
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.dispatchCommand(CraftServer.java:800) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1937) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1776) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1757) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:30) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.util.thread.IAsyncTaskHandler.c(SourceFile:151) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.c(SourceFile:23) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1157) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.MinecraftServer.c(MinecraftServer.java:1) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.util.thread.IAsyncTaskHandler.y(SourceFile:125) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1136) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.MinecraftServer.y(MinecraftServer.java:1129) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.util.thread.IAsyncTaskHandler.bp(SourceFile:110) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1112) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1037) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3332-Spigot-a5dea1c-bc14cb6]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.StringIndexOutOfBoundsException: begin 0, end -1, length 31
        at java.lang.String.checkBoundsBeginEnd(String.java:4601) ~[?:?]
        at java.lang.String.substring(String.java:2704) ~[?:?]
        at dev.array21.skinfixer.commands.subcommands.GetCommand.onSubcommand(GetCommand.java:30) ~[?:?]
        at dev.array21.skinfixer.commands.CommandHandler.onCommand(CommandHandler.java:75) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
        ... 21 more
>
```

**Comment by TheDutchMC** [comment]
That is interesting! I'll dig into this.

Thanks for reporting!

**Comment by TheDutchMC** [comment]
Fixed in v1.7.3
